### DISCUSSION TITLE
List of fixes for the next LTS release candidate

### DISCUSSION THREAD

**Discussion by rene** [lf-edge/eve#4356] [General]
Let's keep track of all desired PRs/commits to be on the next LTS release candidate.

**Comment by rene** [comment]
PR https://github.com/lf-edge/eve/pull/4346

Required for OnLogic FR201 device.

**Reply by OhmSpectator** [reply]
done

**Comment by OhmSpectator** [comment]
https://github.com/lf-edge/eve/pull/4352- EdgeView fix

**Reply by OhmSpectator** [reply]
https://github.com/lf-edge/eve/pull/4374

**Reply by OhmSpectator** [reply]
done

**Comment by OhmSpectator** [comment]
https://github.com/lf-edge/eve/pull/4330 - QMP retry logic fix

**Reply by OhmSpectator** [reply]
https://github.com/lf-edge/eve/pull/4379

**Reply by OhmSpectator** [reply]
done

**Comment by OhmSpectator** [comment]
https://github.com/lf-edge/eve/pull/4357 - memory-monitor cleanup fix

**Reply by OhmSpectator** [reply]
done

**Comment by eriknordmark** [comment]
The EVE PR which pulls in https://github.com/lf-edge/eve-kernel/pull/160 should be back-ported

**Reply by OhmSpectator** [reply]
https://github.com/lf-edge/eve/pull/4380

**Comment by OhmSpectator** [comment]
https://github.com/lf-edge/eve/pull/4367

**Reply by OhmSpectator** [reply]
done
